added note about 21h2 (#10440)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/vpn/how-to-use-single-sign-on-sso-over-vpn-and-wi-fi-connections.md

A "\*Session" credential implies that it is valid for the current user session.
The credentials are also cleaned up when the WiFi or VPN connection is disconnected.

> [!NOTE]

> > In Windows 10 version 21h2 or newer, "\*Session" credential is not visible in Credential Manager.

Reviewer: ```suggestion
> In Windows 10, version 21h2 and later, the "\*Session" credential is not visible in Credential Manager.
```
